frontend/src/lib/api/getData.ts

```diff
@@ -59,7 +55,7 @@ export const getElection = ({
 } = {}): Promise<ElectionProps> => {
   locale ??= currentLocale.get();
   // Match locale softly
-  const matchingLocale = matchLocale(locale, locales.get());
+  const matchingLocale = matchLocale(locale || '', locales.get());
```

Comment:
To me the compiler was complaining that locale could be undefined, since the promise might not return a string, so this was to get rid of the compiler errors :D